### ISSUE TITLE
Action for some/all duplicates

### ISSUE THREAD

**Issue by joram-berger** [cadeyrn/bookmarks-organizer#38]
While all broken links can be deleted at once, the duplicate finder has no bulk action.
I could imagine to prefer the bookmarks in one folder over the other (e.g. if there are duplicates keep the ones in "toolbar" and drop the ones in "my old bookmarks").
In addition one could want to always keep the newer ones and drop the older bookmarks.

**Comment by cadeyrn** [comment]
Thank you for your suggestions.

**Comment by keunes** [comment]
Yes, checkboxes for each bookmark + action 'delete' (and possibly other options) would be great!

**Comment by jjgarcia3843** [comment]
I would love to be able to select several bookmarks to delete, instead of only one at a time. One at a time is not productive. Thanks.

**Comment by gregger404** [comment]
The only reason I tried this app was to get rid of duplicates. Without a bulk action I have no use for it. 

This feature is really important since I use Chrome, Edge, and Firefox on multiple machines and a few times the bookmarks were duplicated so that now I have a tremendous number of duplicates.

If you add the remove bulk option consider making is easy to do this for multiple browsers (or at least give instructions) so that one can clean up all the browsers.

Better yet, how about something that would keep them in sync as an add-on.

thanks!

**Comment by cadeyrn** [comment]
> If you add the remove bulk option consider making is easy to do this for multiple browsers (or at least give instructions) so that one can clean up all the browsers.

A Firefox extension is not able to remove or change anything in a different browser. And giving instructions for other browsers is not in scope for this Firefox extension.

> Better yet, how about something that would keep them in sync as an add-on.

This is also not in scope for this extension and nothing I will develop.

**Comment by gregger404** [comment]
Thank you for your work and quick response!

**Comment by Thomas-D-C** [comment]
> Yes, checkboxes for each bookmark + action 'delete' (and possibly other options) would be great!

For info, [Bookmark Dupes](https://addons.mozilla.org/fr/firefox/addon/bookmark-dupes/) is already able to do that.


Martin Vath told me he has no time for Bookmark Dupes, and it's OK :
- for Bookmarks Organizer to include Bookmark Dupes features.
- to look at his code.

Maybe checkboxes could be replaced by radio buttons (checked one telling the bookmark to keep), the 1st of them always being checked, and allowing to sort bookmarks by many criterias ?

**Comment by tsilvs** [comment]
I came here to confirm that this is much needed feature.

For actual UX inspiration you can refer to `czkawka` or `dupeguru`.

**Comment by Thomas-D-C** [comment]
> I came here to confirm that this is much needed feature.

Thank you :-)

> For actual UX inspiration you can refer to `czkawka` or `dupeguru`.

Could you give links, please?
